<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pychmviewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/images.qrc">
    <normaloff>:/images/pychmviewer.png</normaloff>:/images/pychmviewer.png</iconset>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="PyChmTabs" name="WebViewsWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>873</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="settings_SettingsAction"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="file_Open_action"/>
    <addaction name="file_Print_action"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="view_Increase_font_size_action"/>
    <addaction name="view_Decrease_font_size_action"/>
    <addaction name="view_norm_font_size_action"/>
    <addaction name="view_View_HTML_source_action"/>
    <addaction name="separator"/>
    <addaction name="view_Locate_in_contents_action"/>
    <addaction name="separator"/>
    <addaction name="view_Set_encoding_action"/>
   </widget>
   <widget class="QMenu" name="menu_Windows">
    <property name="title">
     <string>&amp;Windows</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Windows"/>
   <addaction name="menu_Settings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolbar">
   <property name="windowTitle">
    <string>general toolbar</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="file_Open_action"/>
   <addaction name="file_Print_action"/>
  </widget>
  <widget class="QToolBar" name="navToolbar">
   <property name="windowTitle">
    <string>navigation toolbar</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="nav_actionHome"/>
   <addaction name="nav_actionBack"/>
   <addaction name="nav_actionForward"/>
  </widget>
  <widget class="QToolBar" name="viewToolbar">
   <property name="windowTitle">
    <string>action toolbar</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="view_Locate_in_contents_action"/>
   <addaction name="view_Increase_font_size_action"/>
   <addaction name="view_Decrease_font_size_action"/>
   <addaction name="view_norm_font_size_action"/>
   <addaction name="view_View_HTML_source_action"/>
   <addaction name="bookmark_AddAction"/>
  </widget>
  <widget class="QStatusBar" name="mainStatusBar"/>
  <widget class="QDockWidget" name="dockTopics">
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="statusTip">
    <string>Topics</string>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="windowTitle">
    <string>Topics</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="dockIndex">
   <property name="statusTip">
    <string>Index</string>
   </property>
   <property name="windowTitle">
    <string>Index</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="dockSearch">
   <property name="statusTip">
    <string>Search</string>
   </property>
   <property name="windowTitle">
    <string>Search</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QDockWidget" name="dockBookmark">
   <property name="statusTip">
    <string>Bookmark</string>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="windowTitle">
    <string>Bookmark</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
  <action name="file_Print_action">
   <property name="icon">
    <iconset resource="resources/images.qrc">
     <normaloff>:/images/icon_print.png</normaloff>:/images/icon_print.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Print...</string>
   </property>
   <property name="iconText">
    <string>Print</string>
   </property>
   <property name="toolTip">
    <string>Print current page</string>
   </property>
   <property name="whatsThis">
    <string>Prints currently opened page.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="file_exit_action">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="iconText">
    <string>Exit</string>
   </property>
   <property name="whatsThis">
    <string>Exits the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="edit_Copy_action">
   <property name="icon">
    <iconset>
     <normaloff>:/images/icon_copy.png</normaloff>:/images/icon_copy.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;opy</string>
   </property>
   <property name="iconText">
    <string>Copy</string>
   </property>
   <property name="whatsThis">
    <string>Copies selected content to clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="edit_SelectAll_action">
   <property name="icon">
    <iconset>
     <normaloff>../../.designer/backup/image8</normaloff>../../.designer/backup/image8</iconset>
   </property>
   <property name="text">
    <string>&amp;Select All</string>
   </property>
   <property name="iconText">
    <string>Paste</string>
   </property>
   <property name="whatsThis">
    <string>Selects everything in the document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="edit_FindAction">
   <property name="icon">
    <iconset>
     <normaloff>:/images/icon_find.png</normaloff>:/images/icon_find.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Find...</string>
   </property>
   <property name="iconText">
    <string>Find</string>
   </property>
   <property name="toolTip">
    <string>Find text in currently opened page</string>
   </property>
   <property name="whatsThis">
    <string>Enters the Find in page mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="file_ExtractCHMAction">
   <property name="text">
    <string>E&amp;xtract CHM content...</string>
   </property>
   <property name="toolTip">
    <string>Extract the CHM content to the directory</string>
   </property>
   <property name="whatsThis">
    <string>Exctracts the CHM archive content to the specified directory. All the files are extracted.</string>
   </property>
  </action>
  <action name="settings_SettingsAction">
   <property name="text">
    <string>&amp;Application settings...</string>
   </property>
   <property name="toolTip">
    <string>Change the application settings</string>
   </property>
   <property name="whatsThis">
    <string>Change the application settings</string>
   </property>
  </action>
  <action name="bookmark_AddAction">
   <property name="icon">
    <iconset resource="resources/images.qrc">
     <normaloff>:/images/icon_add_bookmark.png</normaloff>:/images/icon_add_bookmark.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add bookmark</string>
   </property>
   <property name="toolTip">
    <string>Adds a bookmark for currently opened page</string>
   </property>
   <property name="whatsThis">
    <string>Adds a bookmark for currently opened page. Remembers the opened page, and scroll position. Bookmarks are accessible through Bookmarks menu or tab.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="view_Increase_font_size_action">
   <property name="icon">
    <iconset resource="resources/images.qrc">
     <normaloff>:/images/icon_font_increase.png</normaloff>:/images/icon_font_increase.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Increase font size</string>
   </property>
   <property name="toolTip">
    <string>Increase the font size</string>
   </property>
   <property name="whatsThis">
    <string>Increases the document font size.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="view_Decrease_font_size_action">
   <property name="icon">
    <iconset resource="resources/images.qrc">
     <normaloff>:/images/icon_font_decrease.png</normaloff>:/images/icon_font_decrease.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Decrease font size</string>
   </property>
   <property name="toolTip">
    <string>Decrease the font size</string>
   </property>
   <property name="whatsThis">
    <string>Decreases the document font size.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="view_norm_font_size_action">
   <property name="icon">
    <iconset resource="resources/images.qrc">
     <normaloff>:/images/icon_font_norm.png</normaloff>:/images/icon_font_norm.png</iconset>
   </property>
   <property name="text">
    <string>&amp;normal size font</string>
   </property>
   <property name="toolTip">
    <string>set the font size normal</string>
   </property>
   <property name="whatsThis">
    <string>set the document font size normal.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="view_View_HTML_source_action">
   <property name="icon">
    <iconset resource="resources/images.qrc">
     <normaloff>:/images/icon_view_source.png</normaloff>:/images/icon_view_source.png</iconset>
   </property>
   <property name="text">
    <string>&amp;View HTML source</string>
   </property>
   <property name="toolTip">
    <string>View HTML source of current page</string>
   </property>
   <property name="whatsThis">
    <string>Shows the HTML source of currently opened page</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="view_Toggle_fullscreen_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable &amp;full screen mode</string>
   </property>
   <property name="whatsThis">
    <string>Enters or leaves the fullscreen mode</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="view_Toggle_contents_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable &amp;side window</string>
   </property>
   <property name="whatsThis">
    <string>Shows or hides the size window (with Content, Index, Search and Bookmark tabs)</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="view_Locate_in_contents_action">
   <property name="icon">
    <iconset resource="resources/images.qrc">
     <normaloff>:/images/icon_locate_in_content.png</normaloff>:/images/icon_locate_in_content.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Locate in contents window</string>
   </property>
   <property name="toolTip">
    <string>Locate the current page in contents window</string>
   </property>
   <property name="whatsThis">
    <string>If the current page is present in the Table of Contents, locate it there. </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+</string>
   </property>
  </action>
  <action name="view_Set_encoding_action">
   <property name="text">
    <string>Set &amp;encoding</string>
   </property>
   <property name="whatsThis">
    <string>Changes the current document encoding. </string>
   </property>
  </action>
  <action name="file_Open_action">
   <property name="icon">
    <iconset resource="resources/images.qrc">
     <normaloff>:/images/icon_open_file.png</normaloff>:/images/icon_open_file.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open a CHM file</string>
   </property>
   <property name="whatsThis">
    <string>Opens a new CHM file</string>
   </property>
  </action>
  <action name="nav_actionBack">
   <property name="icon">
    <iconset resource="resources/images.qrc">
     <normaloff>:/images/icon_back.png</normaloff>:/images/icon_back.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="toolTip">
    <string>Navigate back</string>
   </property>
   <property name="whatsThis">
    <string>Navigate back in navigation history</string>
   </property>
  </action>
  <action name="nav_actionForward">
   <property name="icon">
    <iconset resource="resources/images.qrc">
     <normaloff>:/images/icon_forward.png</normaloff>:/images/icon_forward.png</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="toolTip">
    <string>Navigate forward</string>
   </property>
   <property name="whatsThis">
    <string>Navigate forward in navigation history</string>
   </property>
  </action>
  <action name="nav_actionHome">
   <property name="icon">
    <iconset resource="resources/images.qrc">
     <normaloff>:/images/icon_home.png</normaloff>:/images/icon_home.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="toolTip">
    <string>Navigate home</string>
   </property>
   <property name="whatsThis">
    <string>Navigate to the document Home page, as specified in the document.</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PyChmTabs</class>
   <extends>QWidget</extends>
   <header>pychmtabs.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/images.qrc"/>
 </resources>
 <connections/>
</ui>
